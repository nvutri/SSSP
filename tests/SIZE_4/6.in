c random graph n=32 m=128 lo=0 hi=4096 seed=1548533604
p sp 32 128
a 6 30 1318
a 30 7 1043
a 7 9 963
a 9 13 2590
a 13 27 3673
a 27 10 254
a 10 14 2558
a 14 18 618
a 18 31 2313
a 31 17 2827
a 17 2 2894
a 2 21 1642
a 21 25 2021
a 25 26 727
a 26 11 1593
a 11 16 3123
a 16 29 703
a 29 15 1476
a 15 23 441
a 23 3 3508
a 3 5 4056
a 5 19 426
a 19 20 610
a 20 32 917
a 32 4 467
a 4 1 3506
a 1 12 343
a 12 8 2472
a 8 28 3172
a 28 22 467
a 22 24 3447
a 24 6 219
a 26 30 3839
a 22 6 2967
a 2 31 2170
a 25 20 130
a 28 18 1980
a 15 27 3470
a 11 3 1657
a 26 28 1284
a 21 15 731
a 16 3 2468
a 24 28 1302
a 14 27 3144
a 17 12 3095
a 11 3 3638
a 12 17 1308
a 21 16 1553
a 29 7 3466
a 7 22 2031
a 15 9 2540
a 16 26 3078
a 11 18 1901
a 30 31 259
a 14 21 102
a 1 23 2559
a 10 21 1412
a 8 12 1897
a 32 30 2005
a 27 11 3294
a 7 14 3380
a 31 30 1324
a 7 23 1801
a 27 23 482
a 29 13 1795
a 8 28 1027
a 32 12 623
a 21 27 3477
a 18 11 1562
a 17 28 1598
a 13 3 2097
a 32 2 1655
a 27 2 1247
a 6 1 1266
a 18 26 507
a 15 31 1731
a 11 4 1779
a 30 23 2643
a 6 7 2399
a 13 24 1398
a 21 11 366
a 18 4 1418
a 14 5 2909
a 22 5 3693
a 7 10 161
a 23 15 3049
a 3 31 352
a 10 32 3004
a 27 11 2288
a 21 8 3537
a 27 1 2291
a 7 32 3211
a 2 22 2220
a 28 21 4084
a 25 32 2387
a 6 29 1937
a 22 11 890
a 15 21 2912
a 2 30 1460
a 22 11 3585
a 2 20 1736
a 27 21 196
a 29 8 4009
a 23 25 1962
a 26 1 2970
a 1 13 2626
a 4 13 2788
a 31 16 2353
a 19 9 1484
a 19 28 3824
a 20 12 3757
a 26 7 1954
a 16 30 3547
a 32 16 2528
a 5 32 8
a 9 8 2332
a 15 29 3959
a 11 3 2304
a 6 14 2098
a 2 4 54
a 12 30 2645
a 12 2 4006
a 2 19 2536
a 6 27 2013
a 17 22 117
a 19 30 4029
a 3 8 1300
a 27 25 402
