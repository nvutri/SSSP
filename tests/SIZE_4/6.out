1 0
2 4349
3 4315
4 4403
5 5461
6 4243
7 4031
8 2815
9 4994
10 4192
11 2658
12 343
13 2626
14 6341
15 5608
16 5600
17 1651
18 4559
19 5887
20 4651
21 5604
22 1768
23 2559
24 4024
25 4521
26 5066
27 6256
28 3249
29 6180
30 2988
31 3247
32 5469
